content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -10,16 +10,17 @@ further_reading:
       text: 'Content Security Policy'
 ---
 
-If you experience unexpected behavior with Datadog Browser RUM, this guide may help resolve issues quickly. Reach out to [Datadog support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
+If you experience unexpected behavior with Datadog Browser RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog support][1] for further assistance. Datadog recommends regularly updating to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
```

Comment:
```suggestion
If you experience unexpected behavior with Datadog Browser RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
```

The other changes are great. Avoid if at all possible when you can write it some other way: a) referring to the company by name in the docs, and b) "recommending"  things (yes, the word appears a lot in our docs, but it usually doesn't need to -- certainly don't introduce it).